src/js/components/Select/SelectContainer.js

```diff
@@ -390,14 +390,14 @@ const SelectContainer = forwardRef(
               theme={theme}
             />
           )}
-          <OptionsContainer
-            role="listbox"
-            tabIndex="-1"
-            ref={optionsRef}
-            aria-multiselectable={multiple}
-            onMouseMove={() => setKeyboardNavigation(false)}
-          >
-            {options.length > 0 ? (
+          {options.length > 0 ? (
+            <OptionsContainer
```

Comment:
EmptySearchOption is also used in SelectMultipleContainer.js in a similar way. Do you need to pull it out of the container there also?

Reply:
Good call, I moved the EmptySearchOption outside of the options container in SelectMultipleContainer.js